src/lib/theme/theme-documentation/theme-story.scss

```diff
@@ -0,0 +1,23 @@
+.alternate-theme {
+  --primary-color: red;
+  --secondary-color: orange;
+}
+
+.theme-story {
+  &-grid {
+    display: flex;
+  }
+
+  li {
+    margin-right: 8px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    div {
+      border-radius: theme('borderRadius.default');
+      height: 100px;
+      width: 100px;
```

Comment:
Should we go ahead and stick to rule-of-eights here, too? So `96px` or `104px` - and we could use our Tailwind theme sizes here, too.